packages/create-cloudflare/templates/next/workers/templates/public/_headers

```diff
@@ -0,0 +1,3 @@
+# https://developers.cloudflare.com/workers/static-assets/headers
+/_next/static/*
```

Comment:
what if the content changes in a new deployment?

Reply:
Discussed offline with @conico974.
There is a checksum in the path, so a content change would change the path.